health/health.d/anomalies.conf

```diff
@@ -0,0 +1,8 @@
+# raise a warning alarm if an anomaly flag is "on" (anomalies_local.anomaly=1) 
+# for most of a 5 minute interval for any dimension
+
+ alarm: anomaly_flags
+    on: anomalies_local.anomaly
```

Comment:
should be `template` `on` context

`anomalies_local is `moduleName_jobName`. Change name - the alarms doesn't work.

Reply:
```
 template: anomaly_flags
    on: context
```